apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -330,6 +375,33 @@ const aichatSlice = createSlice({
   },
 });
 
+const hasFilledOutModelCard = (modelCardInfo: ModelCardInfo) => {
+  for (const key of Object.keys(modelCardInfo)) {
+    const typedKey = key as keyof ModelCardInfo;
+
+    if (typedKey === 'exampleTopics') {
+      if (
+        !modelCardInfo['exampleTopics'].filter(topic => topic.length).length
+      ) {
+        return false;
+      }
+    } else if (!modelCardInfo[typedKey].length) {
+      return false;
+    }
+  }
+
+  return true;
+};
+
+// Selectors
+const selectModelCardInfo = (state: {aichat: AichatState}) =>
```

Comment:
Does this need to be a separate selector or could this just be declared inline in `selectHasFilledOutModelCard`?